File: BOJ_1233/BOJ_1233.py
import sys
ANS = {}
A, B, C = map(int, sys.stdin.readline().split())
for i in range(1, A+1, 1):
    for j in range(1, B+1, 1):
        for k in range(1, C+1, 1):
            try: ANS[i+j+k] += 1
            except: ANS[i+j+k] = 1
print(max(ANS.keys(), key=(lambda k:ANS[k])))

File: BOJ_1233/BOJ_1233.cs
using System;
using System.Linq;
namespace asharien{
        class PROGRAM{
                static void Main(){
                        int[] X = new int[128];
                        int[] L = Console.ReadLine().Split().Select(int.Parse).ToArray();
                        for(int i=1; i<=L[0]; i++){
                                for(int j=1; j<=L[1]; j++){
                                        for(int k=1; k<=L[2]; k++){
                                                X[i+j+k] += 1;
                                        }
                                }
                        }
                        int ANS = X.Max();
                        int COLL = Array.IndexOf(X, ANS);
                        Console.WriteLine(COLL);

                }
        }
}
